WorkloadManagementSystem/Service/JobStateUpdateHandler.py

```diff
@@ -183,27 +183,38 @@ def export_setJobStatusBulk(self, jobID, statusDict):
     lastTime = Time.toString(Time.fromEpoch(lastTime))
 
     dates = sorted(statusDict)
+    log = self.log.getSubLogger('JobStatusBulk/Job-%s' % jobID)
+    log.verbose("*** New call *** Last update time %s - Sorted new times %s" % (lastTime, dates))
+    # Remove useless items in order to make it simpler later
+    for sDict in statusDict.values():
+      for item in ('Status', 'MinorStatus', 'ApplicationStatus', 'ApplicationCounter'):
+        if not sDict.get(item):
+          sDict.pop(item, None)
     # Pick up start and end times from all updates, if they don't exist
     for date in dates:
       sDict = statusDict[date]
-      status = sDict.get('Status', status)
+      status = sDict.get('Status', '')
       if status in JobStatus.JOB_FINAL_STATES and not endTime:
         endTime = date
-      minor = sDict.get('MinorStatus', minor)
+      minor = sDict.get('MinorStatus', '')
       # Pick up the start date
       if minor == "Application" and status == JobStatus.RUNNING and not startTime:
         startTime = date
 
     # We should only update the status if its time stamp is more recent than the last update
     if dates[-1] >= lastTime:
       # Get the last status values
-      for date in [date for date in dates if date >= lastTime]:
+      for date in [dt for dt in dates if dt >= lastTime]:
         sDict = statusDict[date]
+        log.verbose("\tTime %s - Statuses %s" % (date, str(sDict)))
```

Comment:
Same here. I would prefer `debug`

Reply:
OK, will do... I set it verbose because the service is running in verbose mode ;-) 
`log` is already a sub logger with `jobStatusBulk`...

Reply:
I thought the rule `(<fixed string>, <variable string>)` was only mandatory for levels more severe than INFO?

Reply:
Also for `VERBOSE`.

Reply:
Done anyway